### PR TITLE
Add support for ONNX Crop operator

### PR DESCRIPTION
**Pull request by postrational**
### Details:
Add support for deprecated ONNX Crop operator.

### CODE REVIEW

File: ngraph/frontend/onnx/onnx_import/src/op/crop.cpp [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                    // Input data shape [N,C,H,W], slicing only along spatial dimensions
+                    std::vector<int64_t> begin_mask{1, 1, 0, 0};
+                    std::vector<int64_t> end_mask{1, 1, 0, 0};
+
+                    // Default strides: 1,1,1,1
+                    const auto strides = default_opset::Constant::create(
+                        ngraph::element::i64, Shape{4}, {1, 1, 1, 1});
+
+                    return {std::make_shared<default_opset::StridedSlice>(
+                        input_data, begin, end, strides, begin_mask, end_mask)};
```
Comment (tsocha) [inline ngraph/frontend/onnx/onnx_import/src/op/crop.cpp:87]: Since we can create a StridedSlice without strides input, do we really need it here?

**Review by tsocha — CHANGES_REQUESTED** [review]

File: ngraph/frontend/onnx/onnx_import/src/op/crop.cpp (hunk 1 above)
Comment (postrational) [reply ngraph/frontend/onnx/onnx_import/src/op/crop.cpp]: Done.

**Review by tsocha — APPROVED** [review]

**Comment by azhogov** [comment]
/azp openvino-lin-cc, openvino-ngraph-onnx-lin, openvino-onnxruntime-lin, openvino-win-cc

**Comment by azhogov** [comment]
/azp openvino-lin-cc,openvino-ngraph-onnx-lin,openvino-onnxruntime-lin,openvino-win-cc

**Pull request merged**